src/test/resources/features/3.6.x.feature

```diff
@@ -1,75 +1,75 @@
-Feature: Running integration tests for validate module
+Feature: validate_integration_36 
```

Comment:
@jordanpadams it is absolutelly ok to update them as `_3_6`.

I did not try the option with the . because the dot is already used as a separator between the feature id and the test id to create the testrail id.